art/README.md

```diff
@@ -0,0 +1,52 @@
+# AKRI Logo & Assets
```

Comment:
"Akri" is foremost a proper noun -- means edge in Greek -- and secondly also conveniently and acronym (AKRI). Certainly confusing but possibly twice the fun. Could you update the documentation here to use "Akri"?

Reply:
Thanks @kate-goldenring I've corrected the casing in the readme.